amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -354,7 +354,7 @@ export class TableDetail extends React.Component<TableDetailProps & RouteCompone
                   !data.is_view && <WatermarkLabel watermarks={ data.watermarks }/>
                 }
                 <TableDescEditableText
-                  maxLength={ 750 }
+                  maxLength={ 4000 }
```

Comment:
@verdan, just checking -- does Atlas have any limitation in regards to the character length it can store for the table and column descriptions?

@Mikhail-Ivanov I also think since the change to support markdown is on `EditableText` we should increase the default `maxLength` on `EditableText` instead and then remove this override here. Then we'll have the full benefit of this work on `ColumnDescEditableText` as well. Our users at Lyft have requested a larger character limit for both table and column descriptions.

Reply:
The `maxLength` topic really does get into a subjective space -- I think there will always be users who want to write more, and likewise there might always be users who wish descriptions would have more brevity. I am okay with leaving this as `4000` for now unless this will cause any problems for Atlas.

Perhaps @dwon or I can think a bit more about what is optimal after this is merged. Maybe we will end up limiting the `maxLength` based on it's affect on the UI. For example, there is likely a point where the UI becomes harder to use if the descriptions are too big.

Reply:
You're right, these limits are completely subjective. One way around this would be to define default values (say 2000?) and allow the config files to override. Also worth noting is that table descriptions might need more characters than a column description. I wouldn't block this PR on configuring these values, but it is a nice-to-have feature.